apps/frontend/src/hooks/queries/useRequeteDetails.ts

```diff
@@ -1,19 +1,26 @@
 import { useQuery } from '@tanstack/react-query';
-import { client } from '@/lib/api/hc';
-import { handleRequestErrors } from '@/lib/api/tanstackQuery';
+import { fetchRequeteDetails, fetchRequeteOtherEntitiesAffected } from '@/lib/api/fetchRequetesEntite';
 
 export const useRequeteDetails = (requestId?: string) => {
   return useQuery({
     queryKey: ['requete', requestId],
     queryFn: async () => {
       if (!requestId) return null;
-      const response = await client['requetes-entite'][':id'].$get({
-        param: { id: requestId },
-      });
-      await handleRequestErrors(response);
-      const result = await response.json();
-      return result.data;
+      const response = await fetchRequeteDetails(requestId);
+      return response;
```

Comment:
```suggestion
      return fetchRequeteDetails(requestId);
```
L'intéret de faire deux lignes c'est juste pour ajouter un console.log au milieu